mmv1/templates/terraform/examples/scc_v2_organization_mute_config_basic.tf.erb

```diff
@@ -0,0 +1,11 @@
+resource "google_organization" "default" {
```

Comment:
There is no `google_organization` terraform resource, only a data source. You should use the existing test org for this test.

Reply:
addressed